src/components/Admin/ManageSensors/DeleteOldDataModal.tsx

```diff
@@ -0,0 +1,153 @@
+import {
```

Comment:
There's a weird merge thing where this file exists twice in two different folders

Reply:
Which page is desired?

Reply:
*keep* :  /Admin/ManageSensors/DeleteOldDataModal.tsx
*get rid of* /Admin/UserPage/DeleteOldDataModal.tsx

Reply:
Interestingly I don't see an /Admin/UserPage/DeleteOldDataModal.tsx  in my local version